fetch.bs

```diff
@@ -2498,7 +2508,32 @@ response <a for=/>header</a> can be used to require checking of a <a for=/>respo
 `<code>Content-Type</code>` <a for=/>header</a> against the <a for=request>destination</a> of a
 <a for=/>request</a>.
 
-<p>Its <a for=header>value</a> <a>ABNF</a>:
+<p>To <dfn>determine nosniff</dfn>, given a <a for=/>header list</a> <var>list</var>, run these
+steps:
+
+<ol>
+ <li><p>Let <var>value</var> be the result of <a for="header list">getting</a>
+ `<a http-header><code>X-Content-Type-Options</code></a>` from <var>list</var>.
+
+ <li><p>If <var>value</var> is null, then return false.
+
+ <li><p>Let <var>stringValue</var> be the <a>isomorphic decode</a> of <var>value</var>.
+
+ <li>
+  <p>Let <var>tokens</var> be the result of
+  <a lt="split on commas">spliting <var>stringValue</var> on commas</a>.
+
+  <p class="note">This intentionally strips U+000C FORM FEED, despite 0x0C not being being a
+  <a>HTTP whitespace byte</a>.
```

Comment:
Sorry, hadn't really focused on this.  What's the motivation here?  Historical reasons?  Looks like Chrome manages to not treat this byte as special, at least in the HTTP header parser.  If we don't absolutely need this, I'd rather get rid of it, just from a keeping things consistent standpoint.

Reply:
Implementations are a little divergent here. I guess we can try to push for not stripping U+000C here and elsewhere.

Reply:
I'm rather reluctant to add this to Chrome, since it's inconsistent with handling of other headers, and we've gotten away without it for all these years.

I have a CL in progress to remover single quote handling in our general parser for things like 'foo=bar; baz="Jumbo, shrimp"', which was configurable so it could just recognize double quotes, or double-quotes or single quotes, which will make parsing of headers more consistent.  I think it would be unfortunate if we had to add a way of configuring it to treat U+000C as white space in some, but not all header-related contexts, taking a step in the opposite direction.  I also think treating it as whitespace everywhere is a bigger change to the web platform than allowing it as whitespace nowhere.

Reply:
Yeah, you're probably right. I guess I should just add a second whitespace concept. Thanks!